app/Http/Middleware/SecurityHeaders.php

```diff
@@ -22,6 +23,10 @@ class SecurityHeaders
 
     public function handle($request, Closure $next)
     {
+        if (App::environment(['testing', 'testing-ci'])) {
+            return $next($request);
+        }
+
```

Comment:
Reviewers: double-check that this check makes sense please. We don't want accidentally disable this middleware 😅

Reply:
Why would we want to bypass this in testing? (I'm sure there's a good reason for it, I'm just not coming up with it :) )

Reply:
Great question that made me dig in a little more.

[Calling `header_remove`](https://github.com/snipe/snipe-it/blob/13ed2a05a3b819fd05a328902f197dfe8c822462/app/Http/Middleware/SecurityHeaders.php#L107) in this middleware causes the tests to throw exceptions with the following error:

```
ErrorException : Cannot modify header information - headers already sent by (output started at /path/to/project/vendor/phpunit/phpunit/src/Util/Printer.php:104)
 /path/to/project/app/Http/Middleware/SecurityHeaders.php:112
 /path/to/project/app/Http/Middleware/SecurityHeaders.php:30
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
 /path/to/project/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:21
 /path/to/project/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php:31
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
 /path/to/project/app/Http/Middleware/CheckForDebug.php:25
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
 /path/to/project/app/Http/Middleware/CheckForSetup.php:25
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
 /path/to/project/vendor/fideloper/proxy/src/TrustProxies.php:57
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
 /path/to/project/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php:49
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
 /path/to/project/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php:121
 /path/to/project/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php:64
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
 /path/to/project/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php:86
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
 /path/to/project/app/Http/Middleware/NoSessionStore.php:28
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
 /path/to/project/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:103
 /path/to/project/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:142
 /path/to/project/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:111
 /path/to/project/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:510
 /path/to/project/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:476
 /path/to/project/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:306
 /path/to/project/tests/Feature/Api/Users/UsersForSelectListTest.php:25
```

That `if` check in such an important middleware is sketchy though. I'm pulling it out of this and adding a comment about it in the PR description.